### PR TITLE
Various small corrections

### PR DESCRIPTION
**Pull request by fiveop**
Each (type of) change has its own commit, so they can be easily be removed, if reviewers disagree.

Each change adds, removes, or changes at most a handful of words.

### CODE REVIEW

File: _episodes/01-welcome.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ organization before starting the training.
 
 Software Carpentry, Data Carpentry, and Library Carpentry
 are official Lesson Programs of The Carpentries.
-Together, they form a **global community of volunteer researchers, educators, and others** oriented around
+Around them, The Carpentries forms a **global community of volunteer researchers, educators, and others** oriented around
```
Comment (brownsarahm) [inline _episodes/01-welcome.md:113]:
```suggestion
The Carpentries is a **global community of volunteer researchers, educators, and others** oriented around
```

The original sentence sort of had historical baggage that doesn't need to be here. We expand on this later, so I think an overall simpler sentence is more clear here.

**Review by brownsarahm — COMMENTED** [review]
Thanks @fiveop these are mostly great! commit my suggestion or iterate further on that one point and then we'll merge

File: _episodes/01-welcome.md (hunk 1 above)
Comment (fiveop) [reply _episodes/01-welcome.md]: I adopted your suggestion. Thanks

**Comment by brownsarahm** [comment]
Thanks @fiveop!

**Pull request merged**